devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -98,6 +98,9 @@ def parse(String description) {
 			Map descMap = zigbee.parseDescriptionAsMap(description)
 			if (descMap.clusterInt == 0x0001 && descMap.commandInt != 0x07 && descMap?.value) {
 				map = getBatteryResult(Integer.parseInt(descMap.value, 16))
+			} else if (descMap?.clusterInt == 0x0500 && descMap.attrInt == 0x0002) {
```

Comment:
nit: null check on `descMap` isn't necessary because if it was it would be needed above